sdk/search/Azure.Search.Documents/tests/Samples/Sample01_HelloWorld.Data.cs

```diff
@@ -5,6 +5,7 @@ namespace Azure.Search.Documents.Tests.Samples
 {
     public partial class HelloWorld
     {
+#if !SNIPPET
```

Comment:
My word...   should we consider breaking this out into a separate class?

Reply:
Moved this to a new class `HelloWorldData` and now we don't need `if !SNIPPET` for `HelloWorldData.CountriesSolrSynonymMap`.